target/linux/generic/hack-5.4/641-sch_cake-fix-IP-protocol-handling-in-the-presence-of.patch

```diff
@@ -1,114 +0,0 @@
-From a00590d570212c3c633bd463cef8ec7377cc7993 Mon Sep 17 00:00:00 2001
-From: Kevin Darbyshire-Bryant <ldir@darbyshire-bryant.me.uk>
-Date: Tue, 30 Jun 2020 12:07:44 +0100
-Subject: [PATCH] sch_cake: fix IP protocol handling in the presence of VLAN
```

Comment:
Why do you remove this patch?

Reply:
@hauke - https://git.openwrt.org/?p=openwrt/staging/ldir.git;a=commit;h=35e23f471431f791ac4314c591125aad5af8f56e

Reply:
It was fixed more generically upstream so no longer required